frontend/src/pages/LandlordPage.tsx

```diff
@@ -32,12 +32,8 @@ const LandlordPage = (): ReactElement => {
   return (
     <div>
       <h1>{`This is dummy text! My current landlordId is ${landlordId}`}</h1>
-      {reviews.map((review) => (
-        <Review
-          overall_rating={review.overall_rating}
-          date={review.date}
-          text={review.text}
-        ></Review>
+      {reviews.map(({ date, text, overall_rating }) => (
+        <Review overall_rating={overall_rating} date={date} text={text} />
```

Comment:
The prop names are the same, it'd be cleaner to not destructure the argument and use the spread operator instead

Reply:
if the argument is (reviewData), then you can do <Review ...reviewData />